src/main/scala/beam/sim/BeamGraphComparator.scala

```diff
@@ -143,25 +143,39 @@ object BeamGraphComparator {
       }) -> f
     }
     //Group chart files by name (2 level group)
-    val chartsGroupedByPrefix: Map[String, Map[String, Array[(String, File)]]] = fileNames.groupBy(_._1).groupBy(f => {
-      val index = f._1.indexOf("_")
-      if(index == -1)
-        f._1
-      else
-        f._1.substring(0,index)
-    })
+    //    val chartsGroupedByPrefix: Map[String, Map[String, Array[(String, File)]]] = fileNames.groupBy(_._1).groupBy(f => {
+    //      val index = f._1.indexOf("_")
+    //      if(index == -1)
+    //        f._1
+    //      else
+    //        f._1.substring(0,index)
+    //    })
+    val chartsGroupedByPrefix: Map[String, Map[String, Array[(String, File)]]] = fileNames.groupBy(_._1).groupBy {
+      case f if f._1.startsWith("rideHail") => "rideHail".capitalize
+      case f if f._1.startsWith("passengerPerTrip") => "passengerPerTrip".capitalize
+      case f if f._1.startsWith("legHistogram") => "legHistogram".capitalize
+      case f if f._1.startsWith("averageTravelTimes") => "averageTravelTimes".capitalize
+      case f if f._1.startsWith("energyUse") => "energyUse".capitalize
+      case f if f._1.startsWith("modeChoice") => "modeChoice".capitalize
+      case f if f._1.startsWith("physsim") => "physsim".capitalize
+      case f if f._1.startsWith("realizedMode") => "realizedMode".capitalize
+      case f if f._1.startsWith("tripHistogram") => "tripHistogram".capitalize
+      case f if f._1.startsWith("freeFlowSpeedDistribution") => "freeFlowSpeedDistribution".capitalize
+      case _ => "misc".capitalize
+    }
     val subGroups = mutable.HashMap.empty[(String ,String), Map[String, Array[(String, File)]]]
     // set priorities for the grouped chart files
     chartsGroupedByPrefix.foreach(gc => {
       if (gc._2.size == 1){
         val key = gc._2.headOption.map(_._1).getOrElse("")
         key match {
-          case "mode_choice" => subGroups.put("P01" -> key,gc._2)
-          case "energy_use" => subGroups.put("P02" -> key,gc._2)
-          case "realized_mode" => subGroups.put("P03" -> key, gc._2)
+          case "modeChoice" => subGroups.put("P01" -> key,gc._2)
+          case "energyUse" => subGroups.put("P02" -> key,gc._2)
+          case "realizedMode" => subGroups.put("P03" -> key, gc._2)
+          case "misc" => subGroups.put("P99" -> key, gc._2)
```

Comment:
Actually Rashid wanted modeChoice energyUse and reliazedMode charts to take the top priority in the html page . So I had to assign the priority for the groups @JustinPihony

Reply:
I was more referring to the P04 vs P99 change

Reply:
modechoice as 1 , energyuse as 2 , realizedmode as 3 priority was a specific requirement , all other graph prefixes can follow alphabetical order (so all of them will can have same priority level -  4) , while the "misc" group can always go last.
I gave 99 because if it is given 05 now , in future if some priority charts get added after 03, we need to update misc priority by +1 all the time @JustinPihony .

Reply:
Yes, but now everything **except** `misc` will be tagged as a 4 - thus locking them in. Isn't that just the problem you are trying to avoid? Maybe choosing a middle ground of 50 would work, as it seems sufficiently far off, but before 99? Or even 98?

Reply:
ya 50 would also suffice @JustinPihony  . All I wanted to do was to make sure that **Misc** group comes at the end of the list , so wanted to assign a priority greater(a little far off ) than 4 .